studio/src/components/layout/graph-layout.tsx

```diff
@@ -227,6 +231,18 @@ export const GraphSelect = () => {
       value={selected?.id}
       onValueChange={(gID) => {
         const graph = data?.graphs.find((g) => g.id === gID);
+
+        // don't show subgraphs and compositions page for monographs
+        if (
+          graph?.type === "graph" &&
```

Comment:
We should rename `graph` to `monograph` to make it very explicit.

Reply:
Composition page is still valuable for monographs. Also the recent added feature "Config Signing & Validation" requires it which works with both graph types.

Reply:
Thanks, I did make the page visible but forgot about this redirect. Done.